fdbcli/BulkLoadCommand.actor.cpp

```diff
@@ -185,14 +195,116 @@ ACTOR Future<UID> bulkLoadCommandActor(Reference<IClusterConnectionRecord> clust
+		wait(submitBulkLoadJob(cx, bulkLoadJob));
+		return bulkLoadJob.getJobId();
+
+	} else if (tokencmp(tokens[1], "blobstore")) {
+		if (tokens.size() != 6) {
+			printUsage(tokens[0]);
+			return UID();
+		}
+		UID jobId = UID::fromString(tokens[2].toString());
+		Key rangeBegin = tokens[3];
+		Key rangeEnd = tokens[4];
+		// Bulk load can only inject data to normal key space, aka "" ~ \xff
+		if (rangeBegin >= rangeEnd || rangeEnd > normalKeys.end) {
+			printUsage(tokens[0]);
+			return UID();
+		}
+		std::string jobRoot = tokens[5].toString();
+		KeyRange range = Standalone(KeyRangeRef(rangeBegin, rangeEnd));
+		BulkLoadJobState bulkLoadJob = createBulkLoadJob(jobId, range, jobRoot, BulkLoadTransportMethod::BLOBSTORE);
+		wait(submitBulkLoadJob(cx, bulkLoadJob));
+		return bulkLoadJob.getJobId();
+
+	} else if (tokencmp(tokens[1], "clear")) {
+		if (tokens.size() != 3) {
+			printUsage(tokens[0]);
+			return UID();
+		}
+		state UID jobId = UID::fromString(tokens[2].toString());
+		wait(clearBulkLoadJob(cx, jobId));
+		fmt::println("Job {} has been cleared. No task will be spawned.", jobId.toString());
+		return UID();
+
+	} else if (tokencmp(tokens[1], "status")) {
+		if (tokens.size() != 4) {
+			printUsage(tokens[0]);
+			return UID();
+		}
+		bool anyJob = wait(getOngoingBulkLoadJob(cx));
+		if (!anyJob) {
+			return UID();
+		}
+		Key rangeBegin = tokens[2];
+		Key rangeEnd = tokens[3];
+		if (rangeBegin >= rangeEnd || rangeEnd > normalKeys.end) {
+			printUsage(tokens[0]);
+			return UID();
+		}
+		KeyRange range = Standalone(KeyRangeRef(rangeBegin, rangeEnd));
+		wait(getBulkLoadCompleteRanges(cx, range));
+		return UID();
+
+	} else {
+		printUsage(tokens[0]);
+		return UID();
+	}
+}
+
 CommandFactory bulkLoadFactory(
     "bulkload",
-    CommandHelp("bulkload [mode|acknowledge|local|status] [ARGs]",
-                "bulkload commands",
-                "To set bulkLoad mode: `bulkload mode [on|off]'\n"
-                "To acknowledge completed tasks within a range: `bulkload acknowledge <TaskID> <BeginKey> <EndKey>'\n"
-                "To trigger a task injecting a SST file from local file system: `bulkload local <BeginKey> <EndKey> "
-                "<Folder> <DataFile> <ByteSampleFile>'\n"
-                "To get progress of tasks within a range: `bulkload status <BeginKey> <EndKey> "
-                "[all|submitted|triggered|running|complete] <limit>'\n"));
+    CommandHelp(
+        "bulkload [mode|acknowledge|local|status] [ARGs]",
+        "bulkload commands",
+        "To set bulkload mode: `bulkload mode [on|off]'\n"
+        "To load a range from SST files: `bulkload [local|blobstore] <BeginKey> <EndKey> dumpFolder`\n"
+        "To clear current bulkload job: `bulkload clear <JobID>`\n"
+        "To get completed bulkload ranges: `bulkload status <BeginKey> <EndKey>`\n"
+        "BulkLoad tasks are operated when setting unittest.\n"
```

Comment:
unittest is not mentioned as an option for bulkload but you have help down here on it.

What is acknowledge here?

Should blobstore be mentioned if local is?